packages/app/src/systems/Asset/components/AssetItem/AssetItem.tsx

```diff
@@ -108,17 +108,22 @@ export const AssetItem: AssetItemComponent = ({
     navigate(Pages.assetsEdit({ id: asset.assetId }));
   }
 
+  // Using memo to avoid avatar of token flash when component rerender
+  const MemoAvatar = useMemo(() => {
+    return imageUrl ? (
+      <Avatar
+        name={name}
+        src={imageUrl}
+        css={{ height: 36, width: 36, borderRadius: '$full' }}
+      />
+    ) : (
+      <Avatar.Generated hash={assetId} css={{ height: 36, width: 36 }} />
+    );
+  }, [name, imageUrl, assetId]);
```

Comment:
I've just checked it deeper and noticed not only these two componentes are re-rendering.

<img width="392" alt="Screenshot 2024-03-13 at 09 45 30" src="https://github.com/FuelLabs/fuels-wallet/assets/7074983/5c4daa44-6f78-412f-8032-01858e3b270c">

The issue is that `AssetList` is being re-render due to its parent re-render.
And, because of that, `AssetItem` as well.


Could you revert this change and wrap `AssetList` with a `memo`?

e.g.
```tsx
export const AssetList = memo(AssetListBase);
```

With this change I believe you'll also need to replace `AssetList.Loading` with `AssetListLoading`, for example.

Reply:
I forgot one small detail, to make it works, we'll need also to memoize an object on `Home`

https://github.com/FuelLabs/fuels-wallet/blob/7dde82f84a8397f79485607886e4b362f0f22698/packages/app/src/systems/Home/pages/Home/Home.tsx#L51-L53

```tsx
const emptyProps = useMemo(() => {
  return {
    showFaucet: selectedNetwork?.url === VITE_FUEL_PROVIDER_URL,
  };
}, [selectedNetwork]);
```